routes/router.js

```diff
@@ -86,6 +86,7 @@ exports.annotations.saveMutation = annotations.saveMutation;
 exports.annotations.gene = annotations.gene;
 exports.annotations.mutation = annotations.mutation;
 exports.annotations.updateMutation = annotations.updateMutation;
+exports.annotations.createMutationRef = annotations.createMutationRef;
```

Comment:
Most of the routes through annotation have been removed, the exception being annotations.mutation.gene, which we're leaving as a redirection from the dataset view page.